src/index.css

```diff
@@ -95,17 +114,56 @@ h1 {
   margin-bottom: 1rem;
 }
 
+.record:hover {
+  font-size: 20px;
+  text-align: center;
+  color: #955;
+  margin-left: 25px;
+  margin-right: -25px;
+  border-radius: 50%;
+  box-shadow: 10px 10px 10px 6px #000;
+}
+
 #score-form {
   width: 100%;
   height: auto;
-  display: flex;
-  flex-flow: column;
-  align-items: center;
+  text-align: right;
+}
+
+#scoresTXT {
+  width: 100%;
+  margin-bottom: 20px;
+  border: 0;
+  outline: none;
+  box-shadow: 10px 3px 1px 1px #000;
+  transition: all 1s;
 }
 
-#scoresTXT,
 #names {
-  margin-bottom: 15px;
-  border: #000 solid 2px;
-  box-shadow: 1px 1px 1px 1px #000;
+  width: 100%;
+  margin-bottom: 20px;
+  border: 0;
+  outline: none;
+  box-shadow: 10px 3px 1px 1px #000;
+  transition: all 1s;
+}
+
+#names:focus {
+  font-size: 20px;
+}
+
+#scoresTXT:focus {
+  font-size: 20px;
+}
+
+#RefreshBtn:hover {
+  box-shadow: 3px 3px 3px 3px #000;
+  /* stylelint-disable-next-line csstree/validator */
```

Comment:
- [ ] [OPTIONAL] Kindly fix all linters instead of disabling them, they are in place to help you use best practices